tests/Integration/Behaviour/Features/Scenario/Search/toggle_alias.feature

```diff
@@ -0,0 +1,39 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s alias --tags update_alias_status_feature
+@restore-aliases-before-feature
+@clear-cache-before-feature
+@clear-cache-after-feature
+@update_alias_status_feature
+Feature: Toggle basic alias statuses from Back Office (BO)
+  As a BO user
+  I need to be able to toggle alias statuses from the BO
+
+  Scenario: I disable multiple aliases
+    Given following aliases should exist:
+      | id reference | alias  | search | active |
+      | alias1       | bloose | blouse | 1      |
+      | alias2       | blues  | blouse | 1      |
+    When I disable alias with reference "alias1"
+    And I disable alias with reference "alias2"
+    Then following aliases should exist:
+      | id reference | alias  | search | active |
+      | alias1       | bloose | blouse | 0      |
+      | alias2       | blues  | blouse | 0      |
+
+  Scenario: I enable single alias
+    And I enable alias with reference "alias2"
+    Then following aliases should exist:
+      | id reference | alias  | search | active |
+      | alias1       | bloose | blouse | 0      |
+      | alias2       | blues  | blouse | 1      |
+
+  Scenario: I bulk update alias statuses
+    When I bulk change status to be enabled for following aliases "alias1,alias2"
+    Then following aliases should exist:
+      | id reference | alias  | search | active |
+      | alias1       | bloose | blouse | 1      |
+      | alias2       | blues  | blouse | 1      |
+    When I bulk change status to be disabled for following aliases "alias1,alias2"
```

Comment:
you can find some examples of regex step definition to accept (enable|disable) in same step and convert it to bool
```suggestion
    When I bulk disable following aliases "alias1,alias2"
```